src/Tizen.NUI.Wearable/src/public/NUIWatchApplication.cs

```diff
@@ -25,26 +25,31 @@ namespace Tizen.NUI
     /// <summary>
     /// Represents an application that can make watch-face.
     /// </summary>
+    [Obsolete("This has been deprecated in API12")]
     public class NUIWatchApplication : CoreApplication
     {
         /// <summary>
         /// Occurs whenever the application is resumed.
         /// </summary>
+        [Obsolete("This has been deprecated in API12")]
         public event EventHandler Resumed;
 
         /// <summary>
         /// Occurs whenever the application is paused.
         /// </summary>
+        [Obsolete("This has been deprecated in API12")]
         public event EventHandler Paused;
 
         /// <summary>
         /// Occurs at every second.
         /// </summary>
+        [Obsolete("This has been deprecated in API12")]
         public event EventHandler TimeTick;
 
         /// <summary>
         /// Event arguments that passed via time tick event signal.
         /// </summary>
+        [Obsolete("This has been deprecated in API12")]
         public class TimeTickEventArgs : EventArgs
         {
             private WatchTime _watchTime;
```

Comment:
consider marking those 2 public properties as Obsolete as well, the class is already deprecated, but for consistency members might be as well

Reply:
Thank you for review. 
it has been fixed.